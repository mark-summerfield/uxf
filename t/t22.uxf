uxf 1.0
= Categories CID int Title str Selected bool Date date
(Categories
5 <Classical &amp; &lt;Special&gt;> yes 1999-07-21
4 <Modern Instrumental> no 1976-11-30
1 <New Acquistions> no 1947-02-28
2 <Pop> no 1997-10-01
3 <Punk> no 2002-01-17
7 <Uncategorized> no 2020-02-02
6 <Unpopular Pop> no 2022-12-21
)
