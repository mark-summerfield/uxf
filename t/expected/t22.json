{
  "UXF^table": {
    "name": "Categories",
    "comment": null,
    "fields": {
      "CID": "int",
      "Title": "str",
      "Selected": "bool",
      "Date": "date"
    },
    "records": [
      [
        5,
        "Classical & <Special>",
        true,
        {
          "UXF^date": "1999-07-21"
        }
      ],
      [
        4,
        "Modern Instrumental",
        false,
        {
          "UXF^date": "1976-11-30"
        }
      ],
      [
        1,
        "New Acquistions",
        false,
        {
          "UXF^date": "1947-02-28"
        }
      ],
      [
        2,
        "Pop",
        false,
        {
          "UXF^date": "1997-10-01"
        }
      ],
      [
        3,
        "Punk",
        false,
        {
          "UXF^date": "2002-01-17"
        }
      ],
      [
        7,
        "Uncategorized",
        false,
        {
          "UXF^date": "2020-02-02"
        }
      ],
      [
        6,
        "Unpopular Pop",
        false,
        {
          "UXF^date": "2022-12-21"
        }
      ]
    ]
  }
}